kubernetes.go

```diff
@@ -147,6 +148,15 @@ func (c *KubernetesClient) ExecuteTemplates(info *ClusterInformation) {
 	}
 }
 
+func (c *KubernetesClient) readAnnotation(meta v1.ObjectMeta, annotation string, value interface{}) {
+	if data, ok := meta.Annotations[annotation]; ok && len(data) > 0 {
```

Comment:
I know this is not from this review, but this line is too dense to correctly read it. I would split it like:

```
if data, ok := meta.Annotations[annotation]
if ok && len(data) > 0 {
```

Which reads much better IMO